agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/httpclient/LazyHttpClient.java

```diff
@@ -48,15 +48,14 @@
 import javax.annotation.Nullable;
 import org.checkerframework.checker.lock.qual.GuardedBy;
 import reactor.core.publisher.Mono;
-import reactor.netty.resources.ConnectionProvider;
+import reactor.netty.resources.LoopResources;
 
 public class LazyHttpClient implements HttpClient {
 
   private static final String APPLICATIONINSIGHTS_AUTHENTICATION_SCOPE =
       "https://monitor.azure.com//.default";
 
   private static final HttpClient INSTANCE = new LazyHttpClient();
-  private static final int DEFAULT_MAX_TOTAL_CONNECTIONS = 200;
```

Comment:
200 to 1? what is the motivation?

Reply:
200 is a different number, I don't think we need a non-default value for that.

1 is the number of threads (down from default which is based on cpu-count).

the motivation is in the comment below (to reduce memory consumption, which was a regression in 3.2.0 when we switched http clients)